x-pack/plugins/maps/public/classes/sources/es_agg_source/es_agg_source.ts

```diff
@@ -21,7 +21,7 @@ export const DEFAULT_METRIC = { type: AGG_TYPE.COUNT };
 
 export interface IESAggSource extends IESSource {
   getAggKey(aggType: AGG_TYPE, fieldName: string): string;
-  getAggLabel(aggType: AGG_TYPE, fieldName: string): string;
+  getAggLabel(aggType: AGG_TYPE, fieldLabel: string): string;
```

Comment:
I noticed `AbstractESAggSource` isn't actually implementing this interface. Could you add it to the class-def?

```
export abstract class AbstractESAggSource extends AbstractESSource implements IESAggSource {
```

https://github.com/elastic/kibana/blob/89eadfce365b7ebbeda36e8f35753300da3e74e1/x-pack/plugins/maps/public/classes/sources/es_agg_source/es_agg_source.ts#L31